### PR TITLE
feat: article error state

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Add an error state for native apps with a retry button

screen shot 2018-08-17 at 15 12 17

### CODE REVIEW

File: packages/article/src/article-error.web.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Text, View } from "react-native";
+
+const ArticleError = props => (
+  <View>
+    <Text>An error occurred</Text>
+    <Text>{JSON.stringify(props, null, 2)}</Text>
```
Comment (colincclark) [inline packages/article/src/article-error.web.js]: can we fix this at the same time as doing android?

File: packages/article/src/styles/article-error/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  fontSizes,
+  spacing
+} from "@times-components/styleguide";
+
+export default StyleSheet.create({
+  errorContainer: {
+    height: "100%",
+    justifyContent: "space-between",
+    marginLeft: spacing(2),
+    marginRight: spacing(2),
```
Comment (colincclark) [inline packages/article/src/styles/article-error/index.js]: `marginHorizontal`

File: packages/article/src/styles/article-error/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    paddingTop: spacing(2),
+    paddingBottom: spacing(14)
+  },
+  errorImageContainer: {
+    alignSelf: "center",
+    marginBottom: spacing(4),
+    marginTop: spacing(4)
```
Comment (colincclark) [inline packages/article/src/styles/article-error/index.js]: `marginVertical`

File: packages/article/src/article-error.web.js (hunk 1 above)
Comment (craigbilner) [reply packages/article/src/article-error.web.js]: we should just remove?

Comment (tuncaulubilge) [reply packages/article/src/article-error.web.js]: We are not really using this error state in web, so I just left the error message in here. 

I considered using the same view for web, but we have a local image asset for this view and web doesn't seem to play along with local images.

Comment (tuncaulubilge) [reply packages/article/src/article-error.web.js]: We could also remove it completely, and just return null on error for web

Comment (colincclark) [reply packages/article/src/article-error.web.js]: Yes, remove it completely please

Comment (tuncaulubilge) [reply packages/article/src/article-error.web.js]: Removed in 4a91707

File: packages/article/src/article.web.js [hunk 4]
```diff
@@ -2,7 +2,6 @@ import React from "react";
 import Ad, { AdComposer } from "@times-components/ad";
 import RelatedArticles from "@times-components/related-articles";
 import ArticleBody from "./article-body/article-body";
-import ArticleError from "./article-error";
```
Comment (colincclark) [inline packages/article/src/article.web.js]: can the `.web` file be deleted now?

Comment (tuncaulubilge) [reply packages/article/src/article.web.js]: article-error.web is deleted now

**Pull request merged**